src/plugins/console/server/lib/spec_definitions/json/generated/search_application.list.json

```diff
@@ -1,16 +1,24 @@
 {
   "search_application.list": {
     "url_params": {
+      "error_trace": "__flag__",
+      "filter_path": [],
+      "human": "__flag__",
+      "pretty": "__flag__",
       "q": "",
-      "from": 0,
-      "size": 0
+      "from": "",
+      "size": ""
     },
     "methods": [
       "GET"
     ],
     "patterns": [
       "_application/search_application"
     ],
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/list-search-applications.html"
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/list-search-applications.html",
```

Comment:
Is there a concept of APIs specific to a serverless project type?

Reply:
I don't think so, I asked about that the ES team but they said for now it's just public vs internal vs blocked in serverless. And the ES spec only has `stack` vs `serverless` availability